### DISCUSSION TITLE
Roadmap for Server Side Rendering Caching in Next Js

### DISCUSSION THREAD

**Discussion by suaterkilic** [vercel/next.js#22585] [Help]
I created a Next JS Project and I created REST API with PHP.

I have a website that is constantly dealing with api requests. And I want to cache some locations. I don't want it to constantly request Api.

As an example, all data on My Homepage comes from the REST API.

And it is a bad situation that every time the visitor enters the homepage, they request the REST API again.

For example, I want to clear the cache every 1 hour and request it again. For any data extraction How can I do this with Next Js. What kind of path should I follow?

Thank you. Best regards.

**Comment by stigkj** [comment]
You should look into incremental static re-generation (ISR) which is made for this scenario: https://nextjs.org/docs/basic-features/data-fetching#incremental-static-regeneration.
